collect-structure.rst

```diff
@@ -0,0 +1,98 @@
+***********************************************************
+Structure of ODK Collect directory in Android file system
+***********************************************************
+
+After installation of ODK collect, you can try out the app by :ref:`downloading blank example forms <in-app-get-blank-forms>`, as soon as you download a blank form you would be able see odk directory in the internal storage or external storage depending on the settings of your android device. The structure of the directory is explained below.
+
+.. _directory-structure:
+
+Directory Structure
+----------------------
+
+::
+
+ odk
+ ├── forms
+ │   ├── Sample-form-media
+ │   └── Sample form.xml
+ │  
+ ├── instances
+ │   └── formID_yyyy-mm-dd_hh-MM-ss
+ │       └── formID_yyyy-mm-dd_hh-MM-ss.xml
+ │  
+ ├── layers
+ │   └── Sample layer
+ │        └── file.mbtiles
+ │
+ └── metadata  
+	├── forms.db
+	├── forms.db-journal
+	├── instances.db
+	└── instances.db-journal
+	
+Inside :file:`ODK` directory, there are four subdirectories which are as follows:
+
+.. image:: /img/collect-structure/directory.*
+  :alt: Image showing folders inside odk directory
+  :class: device-screen-vertical
+
+.. _forms-directory:
+
+:guilabel:`forms` Directory
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. image:: /img/collect-structure/sample-form.*
+  :alt: Image showing forms and Sample form-media directory
+  :class: device-screen-vertical
+
+:file:`forms` folder contains all the forms, fetched from aggregate server, also forms designed by users do not have to be uploaded to Aggregate for them to be used. They can be manually added to the forms directory. The :file:`Sample form-media` directory holds :file:`csv` files.
+
+.. tip::
+
+  If your :file:`.csv` files contain very sensitive data, you may not want to upload that data to your server. Instead, you can upload a blank :file:`.csv` file as part of your form, then replace it with the real :file:`.csv` file by hand-copying that file onto each of your devices. You can hand-copy your :file:`.csv` file in media subdirectory associated with your form in this case Sample-form-media directory. That way, sensitive pre-loaded data never passes through the server.
+
+.. _instances-directory:
+
+:guilabel:`instances` Directory
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. image:: /img/collect-structure/instances.*
+  :alt: Image showing folder inside instance directory
+  :class: device-screen-vertical
+  
+.. image:: /img/collect-structure/instances-form.*
+  :alt: Image showing files inside formID_yyyy-mm-dd_hh-MM-ss directory
+  :class: device-screen-vertical
+
+Submissions are located under this directory. This directory will contain subdirectories with names of the form: :file:`formID_yyyy-mm-dd_hh-MM-ss`. Within each of these subdirectories are the submission data file (named: :file:`formID_yyyy-mm-dd_hh-MM-ss.xml`),and associated data files for the images, audio clips, video clips, etc. linked with this submission.
```

Comment:
I would call them "form instances" rather than "submissions".

Also, I think we should say files are "in" a directory, rather than "under" them.

Check for spacing around your comma, and don't use "etc." (see style guide: no latin abbreviations)